pkg/controller/service/clbv1/event_handler.go

```diff
@@ -319,6 +328,16 @@ func (h *enqueueRequestForNodeEvent) enqueueManagedNode(queue workqueue.RateLimi
 	}
 }
 
+func (h *enqueueRequestForNodeEvent) checkServiceAffected(svc *v1.Service) bool {
+	if helper.IsENIBackendType(svc) {
+		return false
+	}
+	if svc.Spec.ExternalTrafficPolicy == v1.ServiceExternalTrafficPolicyTypeCluster {
+		return true
+	}
+	return false
+}
+
```

Comment:
if services in local mode have labels `service.beta.kubernetes.io/alibaba-cloud-loadbalancer-backend-label ` or `service.beta.kubernetes.io/alibaba-cloud-loadbalancer-remove-unscheduled-backend`, they will also been affected.